projects/lights_control/src/lights_config.c

```diff
@@ -0,0 +1,103 @@
+#include "misc.h"
+
+#include "gpio.h"
+#include "lights_config.h"
+#include "lights_events.h"
+
+#define CONVERT_TO_MASK(index) 1 << index
+
+static const GPIOAddress s_board_type_address = { .pin = 13, .port = GPIO_PORT_B };
+
+static const GPIOAddress s_addresses_front[] = {
```

Comment:
Might be easier to read if you split front/back into their own config files.

Reply:
yeah it became neater 👍

Reply:
I meant actual files.